ext/compressors/iaa/iaacodec/include/iaaInterface-c.h

```diff
@@ -0,0 +1,23 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+
+#ifndef IAA_COMPRESS_C_H
```

Comment:
Should we use `#pragma once` here?
We could get rid of:
```
#ifndef IAA_COMPRESS_C_H
#define IAA_COMPRESS_C_H
...
#endif
```

Reply:
Used #pragma once instead of #ifndef IAA_COMPRESS_C_H.